src/core/pageContent/pageContentLayout.module.scss

```diff
@@ -45,19 +45,28 @@
       flex: calc((100% / 3) * 2) 0 1;
       max-width: calc((100% / 3) * 2);
       // TODO: Replace with flex-gap when browsers support it
-      margin-right: $spacing-xs;
+      margin-right: 22px;
     }
   }
 
   & aside {
     @media screen and (min-width: breakpoints.$breakpoint-m) {
       flex: calc(100% / 3) 0 0;
       // TODO: Replace with flex-gap when browsers support it
-      margin-left: $spacing-xs;
+      margin-left: 22px;
     }
   }
 }
 
+.collectionsWrapper {
+  background-color: var(--color-black-5);
+  padding-bottom: $spacing-layout-xl;
+}
+
+.collectionKorosTop {
+  fill: var(--color-white);
```

Comment:
Should this be some kind of korosTopColor -variable? The app that is using this could use a dark theme for an example.